connexion/api.py

```diff
@@ -181,10 +183,8 @@ def add_paths(self, paths=None):
                     if self.debug:
                         logger.exception(error_msg)
                     else:
-                        import sys
                         logger.error(error_msg)
-                        et, ei, tb = sys.exc_info()
-                        raise ei.with_traceback(tb)
+                        six.reraise(*sys.exc_info())
```

Comment:
There is a PR hanging for that https://github.com/zalando/connexion/pull/162
But I am willing to accept this here since there is no activity, from the author, in that PR for quite some time.